packages/sdk/client/src/halo/halo-proxy.ts

```diff
@@ -105,6 +105,7 @@ export class HaloProxy implements Halo {
     const identityStream = this._serviceProvider.services.IdentityService.queryIdentity();
     identityStream.subscribe((data) => {
       // Set tracing identity. For early stage debugging.
+      // TODO(nf): make the telemetry subscribe to identity changes to set the Sentry user/tags rather than relying on tracing to set them
```

Comment:
Ideally we'd be using something less identifiable than the identity key since it would allow people to leave telemetry on without identifying themselves. We experimented with a pseudononymous id a bit but it needs to be stored in halo in order for it to have the same effect and the halo data space is not yet exposed.

Reply:
that's a good callout, will think about that one a bit as we think about then next round of HALO features.

my hope with this was some fairly quick way to be able to distinguish and identify our own usage

we probably some way to opt out completely as we're collecting lots of other PII like IP address